qiita_pet/uimodules/preprocessed_data_tab.py

```diff
@@ -40,7 +42,11 @@ def render(self, study_id, preprocessed_data):
         is_local_request = self._is_local()
         show_ebi_btn = user.level == "admin"
         processing_status = preprocessed_data.processing_status
-        processed_data = preprocessed_data.processed_data
+        processed_data = [(id_,
+                           '<br/>'.join(['<b>%s:</b> %s' % (k, v) for k, v in
```

Comment:
minor, but the formatting will implicitly split up the tuple so the expansion of `(k, v)` isn't necessary